bcolz/carray_ext.pyx

```diff
@@ -734,6 +734,19 @@ cdef class chunks(object):
             scomp = schunk.read(ctbytes)
         return scomp
 
+    def __iter__(self):
+       self._iter_count = 0
+       return self
+
+    def __next__(self):
```

Comment:
Maybe this function could do the trick, I'm not sure I never used it before, I'll check it, thanks for the tip!